Dockerfile

```diff
@@ -1,46 +1,55 @@
-FROM ubuntu:focal-20220105
+FROM ubuntu:20.04
 ARG PYTHON_VERSION=3.8
 
 RUN groupadd -g 5000 sync-engine \
   && useradd -d /home/sync-engine -m -u 5000 -g 5000 sync-engine
 
 ENV TZ="Etc/GMT"
-RUN DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get dist-upgrade -y && \
-  apt-get install -y tzdata && \
-  apt-get install -y \
-  build-essential \
-  curl \
-  dnsutils \
-  gcc \
-  g++ \
-  git \
-  python-dev \
-  wget \
-  gettext-base \
-  language-pack-en \
-  libcurl4-openssl-dev \
-  libmysqlclient-dev \
-  libxml2-dev \
-  libxslt-dev \
-  libxslt1-dev \
-  mysql-client \
-  pkg-config \
-  lsof \
-  net-tools \
-  shared-mime-info \
-  telnet \
-  vim \
-  libffi-dev \
-  software-properties-common \
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update \
+  && apt-get dist-upgrade -y \
+  && apt-get install -y \
+    tzdata \
+    build-essential \
+    curl \
+    dnsutils \
+    gcc \
+    g++ \
+    git \
+    python-dev \
+    wget \
+    gettext-base \
+    language-pack-en \
+    libcurl4-openssl-dev \
+    libmysqlclient-dev \
+    libxml2-dev \
+    libxslt-dev \
+    libxslt1-dev \
+    mysql-client \
+    pkg-config \
+    lsof \
+    net-tools \
+    shared-mime-info \
+    telnet \
+    vim \
+    libffi-dev \
+    software-properties-common \
   && rm -rf /var/lib/apt/lists/*
 
 RUN if [ "${PYTHON_VERSION}" != "3.8" ] ; \
   then \
     add-apt-repository ppa:deadsnakes/ppa; \
-  fi; \
-  DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y python"${PYTHON_VERSION}"-dev; \
-  if [ "${PYTHON_VERSION}" = "3.8" ] || [ "${PYTHON_VERSION}" = "3.9" ] ; then DEBIAN_FRONTEND=noninteractive apt-get install -y python"${PYTHON_VERSION}"-distutils; fi; \
-  rm -rf /var/lib/apt/lists/*
+  fi \
+  && apt-get update \
+  && apt-get install -y python"${PYTHON_VERSION}"-dev \
+  && if [ "${PYTHON_VERSION}" = "3.7" ] \
```

Comment:
3.7 also requires python3.7-distutils to work.

Reply:
can we remove the if condition then? What do we support more than 3.7 / 3.8 / 3.9?

Reply:
docker-compose.yml devtools service [references circle/python:3.6 image](https://github.com/closeio/sync-engine/blob/master/docker-compose.yml#L42), though if that works with 3.7, we can just move that up and get rid of the `if` indeed.

Reply:
Yep, we should do that. No need to support 3.6

Reply:
Done in f99292a83a7cc04eac96b515c5f2d5a158280b2d.